app/scripts/lib/strings.js

```diff
@@ -20,6 +20,8 @@ function () {
 
   t('Sign up to learn more');
 
+  t('Firefox is coming soon to iOS!');
```

Comment:
@shane-tomlinson Do we need this string here if we have `{{#t}}Firefox is coming soon to iOS!{{/t}}` in **app/scripts/templates/marketing_snippet_ios.mustache:7** below?